ui/app/mainui/activitycenter/views/ActivityNotificationMention.qml

```diff
@@ -9,7 +9,47 @@ import shared 1.0
 import utils 1.0
 import shared.panels.chat 1.0
 
+import "../controls"
+
 ActivityNotificationMessage {
     id: root
 
+    badge: notification.message.communityId ? communityBadgeComponent : notification.chatId ? groupChatBadgeComponent : null
+
+    Component {
+        id: communityBadgeComponent
+
+        CommunityBadge {
+            id: communityBadge
+
+            property var community: root.store.getCommunityDetailsAsJson(notification.message.communityId)
+            // TODO: here i need chanel
+            // property var channel: root.store.chatSectionModule.getItemAsJson(notification.chatId)
```

Comment:
Help wanted! Is there an API to fetch channel name and color using chatId?

Reply:
This thing is quite blocking, created new issue for that: https://github.com/status-im/status-desktop/issues/7529